### ISSUE TITLE
Chicago: eventparticipants are not resolving due to trailing whitespace

### ISSUE THREAD

**Issue by derekeder** [opencivicdata/scrapers-us-municipal#375]
There are about 2,700 `eventparticipant`s that do not resolve their OCD IDs due to trailing whitespace: https://puddle.bunkum.us/chicago_council-2c401e3/eventparticipant?_sort=id&entity_type__exact=person&person_id__isnull=1

Ran this update query locally to apply the trim. This caught all of them except for "Cochran, Willie"

```sql
update opencivicdata_eventparticipant set person_id = opencivicdata_person.id 
from opencivicdata_person 
where opencivicdata_person.name = trim(opencivicdata_eventparticipant.name) 
and person_id is null 
and entity_type = 'person';
```

**Comment by derekeder** [comment]
related to https://github.com/datamade/chi-councilmatic/pull/308

**Comment by derekeder** [comment]
closed via [e2ec285](https://github.com/opencivicdata/scrapers-us-municipal/commit/e2ec285e4113d13c087f7f6171f501562090147d)
